packages/neuron-ui/.env

```diff
@@ -1 +1,2 @@
 REACT_APP_NETWORKS=[{"name":"Testnet","remote":"http://localhost:8114"},{"name":"Localhost","remote":"http://localhost:8114"}]
+REACT_EXPLORER=http://localhost:3000
```

Comment:
Please move both `REACT_APP_NETWORKS` and `REACT_EXPLORER` to in-code constants (and update names as well). For the foreseen period there's no need to make it so flexible to be configurable via ENV.

Reply:
REACT_APP_NETWORKS will be removed by another PR which move network management to neuron-wallet.

REACT_EXPLORER will be removed soon.

Reply:
updated in https://github.com/nervosnetwork/neuron/pull/166/commits/ba8fae7ebe7c88d03cad0879d52c0b7071b012f3